From 8be365b35f2e7ee9cf4c3bbfe8b60015ec6d4bdb Mon Sep 17 00:00:00 2001
From: Himanshu Joshi <himajosh@codeaurora.org>
Date: Sat, 2 May 2020 23:41:16 +0530
Subject: [PATCH] net: ipv6: Add MAP-E enable/disable config.

This patch adds Kconfig option for enable/disable MAP-E config.

Signed-off-by: Himanshu Joshi <himajosh@codeaurora.org>
Change-Id: Ib4bcabd0b626da562dc073326cb3e59cfa5e026a
---
 net/ipv6/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/net/ipv6/Kconfig b/net/ipv6/Kconfig
index 658bfed1df8b..aabcd6931a45 100644
--- a/net/ipv6/Kconfig
+++ b/net/ipv6/Kconfig
@@ -340,4 +340,14 @@ config IPV6_IOAM6_LWTUNNEL
 
 	  If unsure, say N.
 
+config MAP_E_SUPPORT
+	bool "IPv6: MAP (RFC 7597) support"
+	default n
+	depends on IPV6
+	help
+	  MAP-E is a mechanism for transporting IPv4 packets across
+	  an IPv6 network using IP encapsulation.
+
+	  If unsure, say N.
+
 endif # IPV6
-- 
2.34.1

